internal/service/alarms/internal/alertmanager/alertmanager.go

```diff
@@ -0,0 +1,147 @@
+package alertmanager
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"text/template"
+
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	namespace  = "open-cluster-management-observability"
+	secretName = "alertmanager-config"
+	secretKey  = "alertmanager.yaml"
+
+	URL          = "http://alarm-server.oran-o2ims.svc.cluster.local:%s/internal/v1/caas-alerts/alertmanager"
+	templateFile = "./internal/service/alarms/internal/alertmanager/alertmanager.tmpl"
```

Comment:
This will only work when running locally as a standalone binary.   This would have to be packaged into our image.

Reply:
Indeed. I changed this part to make it work for both

Reply:
Thanks. I'll have a look at it

Reply:
updated